arXiv.org.js

```diff
@@ -82,23 +82,26 @@ function doWeb(doc, url) {
 	}
 }
 
-var ns = {
-	oai_dc: 'http://www.openarchives.org/OAI/2.0/oai_dc/',
-	dc: 'http://purl.org/dc/elements/1.1/',
-	xsi: 'http://www.w3.org/2001/XMLSchema-instance',
-	n: 'http://www.openarchives.org/OAI/2.0/' // Default
-};
+
 
 function parseXML(text) {
 	//Z.debug(text);
+	
+	var ns = {
+		oai_dc: 'http://www.openarchives.org/OAI/2.0/oai_dc/',
+		dc: 'http://purl.org/dc/elements/1.1/',
+		xsi: 'http://www.w3.org/2001/XMLSchema-instance',
+		n: 'http://www.openarchives.org/OAI/2.0/' // Default
+	};
```

Comment:
Before that move I encountered an error with multiple items:
```
Error: NamespaceError: An attempt was made to create or change an object in a way which is incorrect with regard to namespaces
```
If the variable `ns` is not global anymore, this seems to fix this error.